src/main/java/seedu/data/ingredient/Ingredient.java

```diff
@@ -18,6 +18,6 @@ public Ingredient(String ingredientName, LocalDate expiryDate) {
 
     @Override
     public String toString() {
-        return ingredientName + " | " + expiryDate;
+        return ingredientName + " | " + expiryDate + "\n";
```

Comment:
Should the string version of the ingredient have an endline? Other strings don't normally have an endline unless `System.out.println` attaches it to them.

Reply:
then I think I put the end line infornt of the seperator line then, otherwise the seperator line always ends right beind the msgs cut now we're not spamming `System.out.println` anymore